core/src/toga/fonts.py

```diff
@@ -55,51 +66,45 @@ def __str__(self) -> str:
         return f"{self.family} {size}{weight}{variant}{style}"
 
     @staticmethod
-    def register(family, path, weight=NORMAL, style=NORMAL, variant=NORMAL):
-        """Registers a file-based font with its family name, style, variant
-        and weight. When invalid values for style, variant or weight are passed,
-        ``NORMAL`` will be used.
+    def register(family, path, *, weight=NORMAL, style=NORMAL, variant=NORMAL):
+        """Registers a file-based font.
+
+        **Note:** This is not currently supported on macOS or iOS.
```

Comment:
Any reason to call it out here, rather than in the Notes section of the docs?

Reply:
Where a platform limitation affects only a specific method, I think it's more useful to put the note in that method's documentation so the user will see it at the right time. There's some precedent for this in on the WebView page.

This is especially the case now that `Font.register` is linked from the `font_family` style property. Anyone following that link is unlikely to see the Notes section before they try to use the method.

In fact, for the same reason I've now moved the notes about obique and small caps under their respective style properties, which are linked from the arguments of the Font APIs.

Reply:
It occurred to me that the underlying problem here is that the bulk of the documentation for Font *isn't* in the usage guide for Font - in fact, there *is* no usage guide for font at present. I've pushed an update that reorganises this documentation.